src/main/java/dk/cngroup/lentils/logger/MessageFactory.java

```diff
@@ -1,6 +1,6 @@
 package dk.cngroup.lentils.logger;
 
-import dk.cngroup.lentils.dto.CodewordFormDTO;
```

Comment:
Hmm, dívám se do masteru... Je vůbec potřeba přidávat nějaké další DTO?
Vidím, že metoda verifyCypher už tam jedno má a tohle právě obsahuje field pro ten výsledek.
Nefixnul to už někdo v minulosti?

Reply:
vypada to ze jo :O 
tim padem tam necham stare DTO a upravim jen par malickosti :) 
to je krasna úrace